main.go

```diff
@@ -395,23 +430,41 @@ func main() {
 
 		log.Infof("Zipalign Build Artifact")
 		signedArtifactName := fmt.Sprintf("%s-bitrise-signed%s", buildArtifactBasename, artifactExt)
-		signedBuildArtifactPaths[i] = filepath.Join(buildArtifactDir, signedArtifactName)
-		if err := zipalignBuildArtifact(zipalign, unalignedBuildArtifactPth, signedBuildArtifactPaths[i]); err != nil {
+		fullPath := filepath.Join(buildArtifactDir, signedArtifactName)
+
+		if artifactExt == "aab" {
```

Comment:
ext contains a leading dot, like: `.aab`, pls fix it and make sure we have an integration test for signing an .aab file